app/src/androidTest/java/org/simple/clinic/FakerModule.kt

```diff
@@ -3,12 +3,13 @@ package org.simple.clinic
 import dagger.Module
 import dagger.Provides
 import io.bloco.faker.Faker
-import org.simple.clinic.di.AppScope
+import io.bloco.faker.components.PhoneNumber
 
 @Module
 class FakerModule {
 
+  private val faker = Faker("en-IND")
```

Comment:
Since `TestData` will be used both in the instrumented test suite (where we have Dagger) and in the unit test suite (where we don't), we can't inject anything into it.

Another option I can explore is to expose the fake `PhoneNumber` generator via the `TestData` since it currently holds on to an instance of `Faker` as well. That way, we still will have only one instance of `Faker` in the test codebase. What do you think?

Reply:
Please provide the faker in the function itself and maybe scope it so that it's clear that we are looking for a single instance.

Reply:
Changed it to provide the `Faker` instance as a scoped dependency.